pennylane/__init__.py

```diff
@@ -96,14 +97,25 @@ def device(name, *args, **kwargs):
     This function is used to load a particular quantum device,
     which can then be used to construct QNodes.
 
-    PennyLane comes with support for the following two devices:
+    PennyLane comes with support for the following devices:
 
-    * :mod:`'default.qubit' <pennylane.plugins.default_qubit>`: a simple pure
+    * :mod:`'default.qubit' <pennylane.devices.default_qubit>`: a simple
       state simulator of qubit-based quantum circuit architectures.
 
-    * :mod:`'default.gaussian' <pennylane.plugins.default_gaussian>`: a simple simulator
+    * :mod:`'default.gaussian' <pennylane.devices.default_gaussian>`: a simple simulator
       of Gaussian states and operations on continuous-variable circuit architectures.
 
+    * :mod:`'default.qubit.tf' <pennylane.devices.default_qubit.tf>`: a state simulator
+      of qubit-based quantum circuit architectures written in TensorFlow, which allows
+      automatic differentiation through the simulation.
+
```

Comment:
Shall we add the beta devices here?

Reply:
Good question! :thinking:

Reply:
🙂 Could link to the `beta.plugins` module instead of duplicating it?

Reply:
I deleted the beta devices (there is a reason they are in beta) and added default_qubit.autograd.